src/test/java/com/teragrep/pth10/chartTransformationTest.java

```diff
@@ -305,7 +205,6 @@ void endToEnd4Test() {
             Collections.sort(resultList);
 
             assertEquals(expectedValues, resultList);
-            res.printSchema();
             boolean aggregates = this.streamingTestUtil.getCatalystVisitor().getAggregatesUsed();
```

Comment:
I think this might be deprecated?

Reply:
Seems to be still in use in PTH-07, but kind of unnecessary to check inner logic in the chart tests. Decided to remove the aggregation assertions from the chart tests, but kept the function.